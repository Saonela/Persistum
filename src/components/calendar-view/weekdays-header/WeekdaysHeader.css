.weekdays-header {
    display: flex;
    justify-content: space-between;
    padding-top: var(--margin-s);
    margin-top: var(--margin);
    margin-bottom: var(--margin);
    border-top: 1px solid var(--color-border);
    font-size: var(--font-size-m);
}

.weekday {
    width: var(--width-calendar-card);
    text-align: left;
}


@media only screen and (max-width: 1300px) {
    .weekdays-header {
        display: none;
    }
}
