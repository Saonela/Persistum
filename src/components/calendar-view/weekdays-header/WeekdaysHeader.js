import React from "react";
import "./WeekdaysHeader.css"

const WeekdaysHeader = () => (
        <div className="weekdays-header">
            <div className="weekday">Monday</div>
            <div className="weekday">Tuesday</div>
            <div className="weekday">Wednesday</div>
            <div className="weekday">Thursday</div>
            <div className="weekday">Friday</div>
            <div className="weekday">Saturday</div>
            <div className="weekday">Sunday</div>
        </div>
)

export default WeekdaysHeader;
