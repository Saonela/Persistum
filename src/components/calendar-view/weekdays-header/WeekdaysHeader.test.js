import React from "react";
import {render} from "@testing-library/react";
import WeekdaysHeader from "./WeekdaysHeader";

describe('WeekdaysHeader', () => {
    it('should match snapshot', () => {
        const {asFragment} = render(<WeekdaysHeader/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
