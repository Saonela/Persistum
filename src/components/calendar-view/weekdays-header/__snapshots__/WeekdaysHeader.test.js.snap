// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekdaysHeader should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="weekdays-header"
  >
    <div
      class="weekday"
    >
      Monday
    </div>
    <div
      class="weekday"
    >
      Tuesday
    </div>
    <div
      class="weekday"
    >
      Wednesday
    </div>
    <div
      class="weekday"
    >
      Thursday
    </div>
    <div
      class="weekday"
    >
      Friday
    </div>
    <div
      class="weekday"
    >
      Saturday
    </div>
    <div
      class="weekday"
    >
      Sunday
    </div>
  </div>
</DocumentFragment>
`;
